src/css/templates/events.scss

```diff
@@ -1,6 +1,24 @@
 body.events {
-  div.events-extra-content {
-    padding-top: 0px;
-    margin-top: -24px;
+  div.header-strip {
+    margin-bottom: 0px; // to allow for the four-card-row div of featured items
```

Comment:
No units with 0 values. @stevejalim does this project use Styellint? Would be useful to set that up, as it will catch these early. Also, I believe Prettier might even fix this automatically. I guess the other question is, does this project use Prettier?

If you want to add those two, you can use the following to easily add the needed config files etc.

https://github.com/project-calavera/project-calavera

Reply:
Thanks @schalkneethling - the project does use prettier, in its default configuration only, which reformats/indents, etc but doesn't seem to catch this. Good call about stylelint - I've ticketed it as #1666, because it might be a largish diff to fix up any CSS style issues

Reply:
As for the nested divs, I know what you mean. It likely could be done cleaner, but I'm trying to avoid messing with the partials I've drawn out of protocol.mozilla.org so that breaking upgrades to Protocol are easier to handle. If i get time, I can look to do a "div-reduction" sweep of the codebase, ideally...